doc/modules/lda_qda.rst

```diff
@@ -96,7 +96,7 @@ In the case of QDA, there are no assumptions on the covariance matrices
 .. note:: **Relation with Gaussian Naive Bayes**
 
 	  If in the QDA model one assumes that the covariance matrices are diagonal,
-	  then this means that we assume the classes are conditionally independent,
+	  then this means that we assume the input features are conditionally independent given the label,
```

Comment:
How about this:
If in the QDA model one assumes that the covariance matrices are diagonal,
then the inputs are assumed to be conditionally independent given the class,